docs/content/getting-started/installation.md

```diff
@@ -49,6 +49,13 @@ Confirm that tfautomv is properly installed:
 tfautomv -version
 ```
 
+## Terraform version compatibility
+
+`tfautomv` uses your local Terraform installation. The following _Terraform version_ are supported:
+
+- from `0.13.x` to `1.0.x`, only with the `-output=commands` option, since _moved blocks_ are not supported in these versions
+- all Terraform version above `1.1.0`
+
```

Comment:
I think a better place for this info would be before the How-To's in `docs/content/usage/_index.md`, like this:

```markdown
Tfautomv runs `terraform init` and `terraform plan` in the current directory and
analyses Terraform's plan. Based on this analysis, it identifies resources that
have moved in your codebase but not yet in Terraform's state. For each of those
resources, it appends a `moved` block to the `moves.tf` file.

The following versions of Terraform are supported:

- `1.1.x` and above by default
- `0.13.x` and above when using the `-output=commands` flag

Tfautomv is fully provider-agnostic. It works with all Terraform providers.
```